src/components/SolidLoginForm.jsx

```diff
@@ -76,4 +104,22 @@ function SolidLoginForm(props) {
   }
 }
 
+/**
+ * Looks up the IDP of a webId by querying the webId. 
```

Comment:
"WebID" instead of "webId". The variable name is ok.

Reply:
Changed this in [commit ebef5d5](https://github.com/SolidLabResearch/generic-data-viewer/pull/52/commits/ebef5d58ef68fefba790296965cc65bcb7ff06f3)